Userland/DevTools/HackStudio/ToDoEntries.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, Federico Guerinoni <guerinoni.federico@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "ToDoEntries.h"
+
+namespace HackStudio {
+
+ToDoEntries& HackStudio::ToDoEntries::the()
+{
+    static ToDoEntries s_instance;
+    return s_instance;
+}
+
+void ToDoEntries::set_entries(String const& filename, Vector<Cpp::Parser::TodoEntry> const&& entries)
+{
+    m_document_to_entries.set(filename, move(entries));
+    if (on_update)
+        on_update();
+}
+
+Vector<Cpp::Parser::TodoEntry> ToDoEntries::get_entries()
+{
+    Vector<Cpp::Parser::TodoEntry> ret;
+    for (auto& it : m_document_to_entries)
+        for (auto& entry : it.value)
+            ret.append({ entry.content, it.key, entry.line, entry.column });
```

Comment:
Please add curly braces to any if/else/for/etc body with more than one line:

```suggestion
    for (auto& it : m_document_to_entries) {
        for (auto& entry : it.value)
            ret.append({ entry.content, it.key, entry.line, entry.column });
    }
```

Reply:
I can't squash them because I also changed some struct in function parameter